src/swap-eth-to-erc20.js

```diff
@@ -0,0 +1,40 @@
+// This file is used to convert ETH to VSP for testing purpose only.
+
+'use strict'
+const config = require('config')
+require('dotenv').config()
+const Web3 = require('web3')
+const erc20Abi = require('erc-20-abi')
+const BN = require('bn.js')
+const nodeUrl = config.get('nodeUrl')
+const web3 = new Web3(nodeUrl)
+const WETH = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2'
+const VSP = '0x1b40183efb4dd766f11bda7a7c3ad8982e998421'
+const DECIMAL = new BN('1000000000000000000')
+const { getRouterContract } = require('./../packages/erc-20-lib/src/uniswap')
+
+function convertEthToERC20(token, from, amount) {
+  const path = [WETH, token]
+  const amountIn = new BN(amount).mul(DECIMAL).toString()
+  getRouterContract(web3)
+    .methods.swapExactETHForTokens(
+      1,
+      path,
+      from,
+      Math.round(Date.now() / 1000) + 60
+    )
+    .send({ from, value: amountIn, gas: 200000 })
+    .then(function () {
+      const erc20Contract = new web3.eth.Contract(erc20Abi, token)
+      return erc20Contract.methods
+        .balanceOf(from)
+        .call()
+        .then(function (balance) {
+          console.log(`Swap success, ERC20 token balance = ${balance}`)
+        })
+    })
+    .catch(console.error)
+}
+
+// For Test, convert 1000000 ETH to VSP
+convertEthToERC20(VSP, '0x23878197A56BFD9088b196f4b3CeE8D21B621f08', 1000000)
```

Comment:
what is this hard coded address is?

Reply:
This is the sender's address. When fork using the mnemonic, it will be the first address.